docs/manual/common/guide/devmode/KafkaServer.md

```diff
@@ -26,7 +26,7 @@ In sbt:
 
 ## Kafka properties file
 
-The Kafka server can be configured with a property file. By default, we are using the stock `server.properties` file provided with Kafka 0.10, with only one change to allow auto creation of topics on the server. This is a good default to quickly get started, but if you find yourself needing to start Kafka with a different configuration, you can easily do so by adding the following in your build.   
+The Kafka server can be configured with an alternative property file. By default, Lagom development environment uses a stock [kafka-server.properties](https://github.com/lagom/lagom/blob/master/dev/kafka-server/src/main/resources/kafka-server.properties) file provided with Kafka 0.10, with only one change to allow auto creation of topics on the server. This is a good default to quickly get started, but if you find yourself needing to start Kafka with a different configuration, you can easily do so by adding your own Kafka `kafka-server.properties` file to you to your build.
```

Comment:
Hmmm I wonder if we need to update this for Kafka 0.11 🤔

Reply:
Maybe we should not mention the version number. That's irrelevant in this context. What counts is that we have our own file with settings that we believe makes sense for development environment and that the user can override it if they want.

Reply:
Oh I meant that I wonder whether the contents of the default server.properties have changed upstream and we might need to update our copy. It's not really relevant to this pull request, sorry. I added a note for myself to follow up on this.

Reply:
True. I updated the docs to not refer to Kafka version, but indeed we should also check the `server.properties` in 0.11.